action.yml

```diff
@@ -5,6 +5,12 @@ inputs:
   version:
     description: "Version of Kind to use (default v0.11.1)"
     default: "v0.11.1"
+  verbosity:
+    description: "numeric log verbosity, (info = 0, debug = 3, trace = 2147483647)"
```

Comment:
nit: could you please keep the description a full upper-case sentence?

Reply:
What do you think about this new version : `Defines log verbosity with a numeric value, (info = 0, debug = 3, trace = 2147483647)` ?